src/server/game/Entities/Player/PlayerUpdates.cpp

```diff
@@ -0,0 +1,2311 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU
+ * GPL v2 license, you may redistribute it and/or modify it under version 2 of
+ * the License, or (at your option), any later version. Copyright (C) 2008-2016
+ * TrinityCore <http://www.trinitycore.org/> Copyright (C) 2005-2009 MaNGOS
+ * <http://getmangos.com/>
+ */
```

Comment:
Just
```cpp
/*
 * Copyright (C) 2016+ AzerothCore <www.azerothcore.org>, released under GNU AGPL v3 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-AGPL3
 */
```

Reply:
Ah thanks, i just copied the one from Player.cpp since that code shouldve been licensed the same, i will change it.